R/bcdc_search.R

```diff
@@ -83,6 +83,11 @@ bcdc_list_group_records <- function(group) {
   cli <- bcdc_catalogue_client("action/group_show")
 
   r <- cli$get(query = list(id = group, include_datasets = 'true'))
+
+  if (r$status_code == 404){
+    stop("404: URL not found - you may have specified an invalid group?")
```

Comment:
```suggestion
    stop("404: URL not found - you may have specified an invalid group?", call. = FALSE)
```

Reply:
☝️ makes a more concise error message (doesn't say "Error in"..."). Could add to the groups function as well